src/java_bytecode/library/Makefile

```diff
@@ -0,0 +1,26 @@
+all: org.cprover.jar
```

Comment:
1. setting up the java library should be a separate PR.
2. isn't someone else working on modelling the Java standard library?